client/app/components/queries/query-editor.js

```diff
@@ -119,31 +121,43 @@ function queryEditor(QuerySnippet, $timeout) {
 
         const schemaCompleter = {
           getCompletions(state, session, pos, prefix, callback) {
-            if (prefix.length === 0 || !$scope.schema) {
+            // make a variable for the auto completion in the query editor
+            $scope.autoCompleteSchema = $scope.schema; // removeExtraSchemaInfo(
```

Comment:
Could you elaborate what this does?

Reply:
It essentially separates the interface schema (`schema`) and the auto complete schema (`autoCompleteSchema`) to handle UI differences between them.

Reply:
To be able to modify the colums further down below?

If we remove the `[P]` code since #2764 isn't going to merged adding it, then what leaves this PR us with?

Reply:
Given all of this, I'd be okay with closing this PR and closing the issue to port this change upstream. What do you think @jezdez?

Reply:
Yeah, let's revisit this if the follow-up for #2764 is merged.